zerver/tests/test_home.py

```diff
@@ -105,7 +105,7 @@ class HomeTest(ZulipTestCase):
         "login_page",
         "max_avatar_file_size_mib",
         "max_file_upload_size_mib",
-        "max_icon_file_size",
+        "max_icon_file_size_mib",
         "max_logo_file_size",
```

Comment:
I'm pretty sure `max_logo_file_size` should get the same treatment; can you do that while you're at it?

Reply:
Yeah, I was thinking the same while working on this.